twine/repository.py

```diff
@@ -19,10 +19,12 @@
 import requests_toolbelt
 import tqdm
 import urllib3
+from packaging import version
```

Comment:
Huh. This is causing mypy to [fail](https://github.com/pypa/twine/pull/719/checks?check_run_id=1453299717#step:5:11) with:

```
Cannot find implementation or library stub for module named 'packaging'
```

But from what I can tell, `packaging` is properly typed: https://github.com/pypa/packaging/tree/master/packaging.

@di Any thoughts?